### PR TITLE
OLS-906: Benchmarks: documentation

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Benchmarks: documentation

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)

## Related Tickets & Documents

- Related Issue #[OLS-906](https://issues.redhat.com//browse/OLS-906)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -295,6 +295,74 @@ ols/app/endpoints/feedback.py                    45     24    47%   37-39, 51, 6
...
+### `pytest-benchmark` package
+
+Benchmarks are based on `pytest-benchmark` package. To create new benchmark, the regular test function needs
+to be created and `benchmark` parameter (fixture) should be passed to it. Then the benchmark can be initialized
+by calling this fixture, passing tested function/method to it, together with required parameters.
+
+### Basic usage of `benchmark` fixture
+
+```python
+def test_format_attachment_plain_text_format(benchmark):
```
Comment (onmete) [inline CONTRIBUTING.md:319]: For the sake of completeness?

`from ols.src.query_helpers.attachment_appender import format_attachment`

**Comment by tisnik** [comment]
/override "Red Hat Konflux / lightspeed-service-on-pull-request"

**Pull request merged**